helm_deploy/django-app/templates/deployment.yaml

```diff
@@ -34,7 +34,7 @@ spec:
                 name: {{ template "django-app.fullname" . }}
                 key: secretKey
           - name: SERVER_NAME
-            value: {{ default "" .Values.deploy.host | quote }}
+            value: {{required "URL is required" .Values.deploy.host }}
```

Comment:
You should still pipe that through `quote`, or wrap the `{{ tags }}` in literal quotes.

Reply:
Agreed.

Reply:
Used the | quote.